runtime/buffer_benchmark.cc

```diff
@@ -198,4 +202,33 @@ BENCHMARK(BM_for_each_slice)->Args({slice_extent, 4, 4});
 BENCHMARK(BM_for_each_contiguous_slice)->Args({slice_extent, 4, 4});
 BENCHMARK(BM_for_each_slice_hardcoded)->Args({slice_extent, 4, 4});
 
+void memset_slices_multi(void* slice, index_t extent, void* slice2) {
```

Comment:
I think it would be a more useful benchmark to make this a copy instead of memsetting two buffers.

Reply:
Done